lib/whitelistedUrlPrefixes.js

```diff
@@ -27,5 +27,6 @@ module.exports = [
   'https://ads-serve.bravesoftware.com/', // remove this once ads catalog moves to using prod
   'https://pdfjs.robwu.nl/logpdfjs', // allowed because it gets canceled in tracking protection
   'https://publishers-distro.basicattentiontoken.org/',
-  'https://publishers-staging-distro.basicattentiontoken.org/'
+  'https://publishers-staging-distro.basicattentiontoken.org/',
+  'https://clients2.google.com' // allowed because it 307's to https://extensionupdater.brave.com
```

Comment:
extensionupdater.brave.com also needs to be added here.

Reply:
done!